editoast/src/core/simulation.rs

```diff
@@ -219,6 +221,11 @@ impl PhysicsConsistParameters {
             .map(|towed| f64::min(towed.const_gamma, self.traction_engine.const_gamma))
             .unwrap_or_else(|| self.traction_engine.const_gamma)
     }
+
+    pub fn compute_etcs_brake_params(&self) -> Option<EtcsBrakeParams> {
+        // TODO: handle towed rolling-stock when applying ERTMS to that case
```

Comment:
Do we have a follow-up about that? I feel we should probably do that in the same PR because intermediate feature in this case doesn't make sense, isn't it?

Reply:
IMO, this is too tricky to be done in this PR.
1. We don't know what data we will get for ETCS towed RS, especially: will we force providing a max_speed? Should we provide a dedicated etcs_max_speed? (the idea is to prevent propagating illegitimate braking gamma for towed to high speeds, up to the traction engine's max_speed). If I were to choose now, I'd go force max_speed when etcs.
2. We didn't read/search about the merging of etcs_brake_params in documentation (and it's heavy). The conservative way is getting the minimum for all case, but it's a bit more code to merge curves, and it may occupy "useless" extra capacity if the merging is smarter than that.

Possible "middle ground": We may just assert that no etcs value is provided on towed RS to explicitly reject such unhandled case. Would that be OK for you?